pom.xml

```diff
@@ -1,177 +1,225 @@
 <?xml version="1.0"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 
-    <modelVersion>4.0.0</modelVersion>
-
-    <parent>
-        <groupId>org.exist-db</groupId>
-        <artifactId>exist-apps-parent</artifactId>
-        <version>1.8</version>
-        <relativePath />
-    </parent>
+  <modelVersion>4.0.0</modelVersion>
 
+  <parent>
     <groupId>org.exist-db</groupId>
-    <artifactId>exist-documentation</artifactId>
-    <version>4.0.5-SNAPSHOT</version>
-
-    <name>eXist-db Documentation</name>
-    <description>Documentation package for eXist-db</description>
-    <url>https://www.github.com/exist-db/documentation</url>
-    <inceptionYear>2001</inceptionYear>
-
-    <organization>
-        <name>eXist-db</name>
-        <url>http://exist-db.org</url>
-    </organization>
-
-    <licenses>
-        <license>
-            <name>GNU Lesser General Public License, version 2.1</name>
-            <url>http://opensource.org/licenses/LGPL-2.1</url>
-            <distribution>repo</distribution>
-        </license>
-    </licenses>
-
-    <scm>
-        <url>https://www.github.com/exist-db/documentation.git</url>
-        <connection>scm:git:https://www.github.com/exist-db/documentation.git</connection>
-        <developerConnection>scm:git:https://www.github.com/exist-db/documentation.git</developerConnection>
-      <tag>HEAD</tag>
+    <artifactId>exist-apps-parent</artifactId>
+    <version>1.8</version>
+    <relativePath/>
+  </parent>
+
+  <groupId>org.exist-db</groupId>
+  <artifactId>exist-documentation</artifactId>
+  <version>4.0.5-SNAPSHOT</version>
```

Comment:
here we have the same value... we should take this as basis and automagically update the other file(s) ?

Reply:
this is controlled by maven, how to use it for files used during the build, and outside of `src/` still requires tweaking